README.md

```diff
@@ -686,6 +686,14 @@ The YourKit Dependency watches the [YourKit Download Page](https://www.yourkit.c
 uses: docker://ghcr.io/paketo-buildpacks/actions/yourkit-dependency:main
 ```
 
+## Updating Go version
+
+Dependabot could suggest to bump the Go version in some places, but you should make sure that the Go version is the same across the different tools in this repository.
```

Comment:
```suggestion
When it is time to update the Go version in the project, you should make sure that the Go version is the same across the different tools in this repository.
```

I'm trying to recall, but I don't think Dependabot will suggest updating them. In the past, we've initiated this manually. Go supports the most two recent versions, so we try to update as the second version is rolling out of support. That way we let the latest version stabilize a bit.